src/Forms/Shared/Samples/Map/AuthorMap/AuthorMap.xaml.cs

```diff
@@ -328,23 +330,27 @@ private void AddLayer(string layerName, string url)
             Uri layerUri = new Uri(url);
 
             // Create a new map image layer
-            layer = new ArcGISMapImageLayer(layerUri);
-            layer.Name = layerName;
+            layer = new ArcGISMapImageLayer(layerUri)
+            {
+                Name = layerName,
 
-            // Set it 50% opaque, and add it to the map
-            layer.Opacity = 0.5;
+                // Set it 50% opaque, and add it to the map
```

Comment:
Comment got detached from "adding it to the map". I'd avoid object initializer here. What are we gaining from it?